src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+    private double side;
+
+    public Square(String color, double side) {
+        super(color);
+        this.side = side;
+    }
+
+    @Override
+    public double calculateArea() {
+        return (double) Math.round((side * side) * 10) / 10;
```

Comment:
All magic numbers in your code should be constants. The number `10` used here for rounding should be defined as a constant with a clear name indicating its purpose.